Base/BacnetScale.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.IO.BACnet.Serialize;
+
+namespace System.IO.BACnet
+{
+    public class BacnetScale : ASN1.IEncode, ASN1.IDecode
+    {
+        public enum BacnetScaleTypes : byte
```

Comment:
I would make the name shorter. Calling it from outside of this class would require writing ```BacnetScale.BacnetScaleTypes``` which is too long and redundant for my taste. I would go with ```BacnetScale.ValueType``` maybe? Please use singular form for enums unless it has a bits attribute ([Microsoft guideline](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations))